qf_lib/brokers/ib_broker/ib_broker.py

```diff
@@ -256,6 +256,9 @@ def _to_ib_order(self, order: Order):
         tif_str = self._map_to_tif_str(time_in_force)
         ib_order.tif = tif_str
 
+        ib_order.eTradeOnly = ''
+        ib_order.firmQuoteOnly = ''
+
```

Comment:
Let's add comments why we need to set those to empty strings - maybe what are those fields etc